test/Elastic.Apm.Feature.Tests/Features/otel_bridge.feature

```diff
@@ -17,6 +17,7 @@ Feature: OpenTelemetry bridge
     Then Elastic bridged transaction is a root transaction
     # outcome should not be inferred from the lack/presence of errors
     Then Elastic bridged transaction outcome is "unknown"
+	Then Elastic bridged transaction trace id is equal to OTel trace id
```

Comment:
These files are synced from https://github.com/elastic/apm/tree/main/tests/agents/gherkin-specs. We sync these files into every elastic APM Agent repo. So these files are managed in the linked repo and not here.

This means, if we change this, first we'd need to open a PR over in that repo and then it'll be synced into this one.

I just realize if you only look at this repo, you'd have no chance to know this - I'll open a PR and add a readme file to `test/Elastic.Apm.Feature.Tests`. This is totally on us and I'm sorry about this.

On the other hand, I don't think this level of detail needs to be specified. 

So, in order to proceed, I'd suggest to remove the change from this file ([otel_bridge.feature](https://github.com/elastic/apm-agent-dotnet/pull/1882/files#diff-ea36b655c118bd668317b2016c43a4e1b75b4367ba5b84ba9e99d25048b07b1f)) and from [test/Elastic.Apm.Feature.Tests/OpenTelemetryBridgeStepDefinitions.cs](https://github.com/elastic/apm-agent-dotnet/pull/1882/files#diff-4368859739d912974d2225c73bb3bd9c1f06b574ed959fde10244b4d2d944fd3) and proceed with this PR without the gherkin changes.